Dynatrace/dynatrace-operator
# Fixed make bundle target for openshift (#984)

# Description

* Fixed regression from #974 where `make bundle` always generates
  manifests for the kubernetes platform, regardless of chosen `PLATFORM`

## How can this be tested?
Verified that the dynamic make target reference works by setting `PLATFORM` shell var to a non-existent make target, expecting failure.

Ran this again with no overridden `PLATFORM` env var (defaulting the value to `openshift`). 

Running `make bundle`, `make bundle/openshift` and `make bundle/kubernetes` should now call the requisite `make manifest/<platform>` target as expected, and without any duplicate shell code.

## Checklist
